src/main/java/step2/PositiveNumberList.java

```diff
@@ -0,0 +1,22 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PositiveNumberList {
+    private List<PositiveNumber> list;
```

Comment:
클래스명이나 변수명이 특정 구현체의 의존적이기보다는 복수를 나타내는 s를 붙여보는건 어떨까요? 🤔 
```suggestion
public class PositiveNumbers {
    private List<PositiveNumber> values;
```